tests/h/cli/commands/authclient_test.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+import mock
+import pytest
+
+from h import models
+from h.cli.commands import authclient as authclient_cli
+
+
+class TestAddCommand(object):
+
+    def test_it_creates_an_authclient(self, cli, cliconfig, db_session):
+        authclient = self._add_authclient(cli, cliconfig, db_session)
+        assert authclient
+
+    def test_it_prints_the_client_id_and_secret(self, cli, cliconfig, db_session, echo):
```

Comment:
Not sure how important it is to test this, if you really want to then we should do some regex matching, because changing something that isn't important shouldn't fail this test. Also, I think the `click` context is storing stdout, so you can just read it from there instead of asserting that the method has been called.